### ISSUE TITLE
To GET or POST: that is the question

### ISSUE THREAD

**Issue by pajooh** [wildfish/django-star-ratings#43]
Hello
Clicking on on a star, i get 405 method not allowed
i checked the source and found that the ```Rate``` view only has a ```post``` method, but the rate submission is done via a simple ```href``` tag. how could this be possible?

**Comment by rolo** [comment]
Hiya.

There's some javascript which adds handlers to the elements to do a `post`. So my guess would be you don't have the js included, or have something preventing it from running.

Really though we should support this working without js being required. I'll make sure this is on the roadmap to be done before the next release.

**Comment by pajooh** [comment]
i'm loading rating resources in a modal via ajax request, but it seems that the documents selectors did not work in this case!

**Comment by rolo** [comment]
Can you send me your code, or show me it hosted somewhere to take a look?

**Comment by pajooh** [comment]
here is the modal content:
```html
{% load i18n %}
{% load bootstrap %}
{% load ratings %}
{% load static %}
<div class="modal-header">
	<button type="button" class="close" data-dismiss="modal" aria-hidden="true">X</button>
	<h2>{{ object.title}}</h2>
</div>
<div class="modal-body">
	{% ratings object %}
	// other modal stuff
</div>
<script type="text/javascript" src="{% static 'star-ratings/js/dist/star-ratings.min.js' %}"></script>
```
the stylesheet are loaded in the base template,

**Comment by rolo** [comment]
Okay gotchya. In this case, I think it's because the rating starts are being loaded after the base document and scripts are loaded, so the selectors aren't finding your star links to change the behaviour of them. I think this is kind of tied in with having things work without JS. Leave it with me and we'll figure something out for handling that case.

**Comment by OmegaDroid** [comment]
Hi @pajooh we have just released version `0.4.0` to PyPI. 

We have added `bindRatings(elem)` to the js which should be available in your modal, you just need to pass the element object you wish to bind to into it.

**Comment by pajooh** [comment]
Thank you all
